ports/pullywog/port.json

```
@@ -0,0 +1,28 @@
+{
+  "version": 3,
+  "name": "pullywog.zip",
+  "items": [
+    "Pullywog.sh",
+    "pullywog"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Pullywog",
+    "porter": [
+      "Slobters"
+    ],
+    "desc": "Sling yourself through the air by catching flies with your tongue. Eating several things without touching the ground will score you a combo, so try to stay in the air for as long as you can! The timer constantly ticks down; you can eat anything to keep it going a little longer, but watch out for ghosts which are lethal to the touch! You can stop bombs from exploding by eating them before they hit the water.",
+    "desc_md": null,
+    "inst": "Ready to run!",
+    "inst_md": null,
+    "genres": [
+      "arcade"
+    ],
+    "image": null,
+    "rtr": true,
+    "exp": false,
+    "runtime": null,
+    "reqs": [],
+    "arch": [aarch64]
```

strings in json files have to be quoted.
```suggestion
    "arch": ["aarch64"]
```